ui/src/pages/insightDetail/components/eventAggregator/styles.module.less

```diff
@@ -349,13 +346,12 @@
         }
       }
 
-      .events_content_diagnosisContent {
+      .events_content_diagnosisBody {
```

Comment:
Removing `font-size: 14px;` and `line-height: 1.6;` might reduce readability. Consider keeping these for consistent text sizing and spacing.